crates/ruma-common/src/events/room/message/content_serde.rs

```diff
@@ -50,3 +65,372 @@ impl<'de> Deserialize<'de> for MessageType {
         })
     }
 }
+
+/// Helper struct for deserializing `AudioMessageEventContent` with stable and unstable field names.
+///
+/// It's not possible to use the `alias` attribute of serde because of
+/// https://github.com/serde-rs/serde/issues/1504.
+#[derive(Clone, Debug, Deserialize)]
+#[cfg(feature = "unstable-msc3246")]
+pub struct AudioMessageEventContentSerDeHelper {
```

Comment:
Nitpick: This helper is only used for deserialization

```suggestion
pub struct AudioMessageEventContentDeHelper {
```